payload_relay.ino

```diff
@@ -0,0 +1,40 @@
+#if false
+
+#ifndef PAYLOAD_THRESHOLD
+#define PAYLOAD_THRESHOLD 14.5 // height of one floor in Baldwin staircase in units of feet
```

Comment:
I think Riley already said this, but this should prolly be a 'const float' or something